.github/workflows/package.yml

```diff
@@ -45,8 +45,14 @@ jobs:
       run: |
         npm config set "//npm.pkg.github.com/:_authToken" ${{ secrets.GITHUB_TOKEN }}
         npm install
-    - name: npm build
-      run: npm run build
+      # build for home
+    - name: npm build home
+      if: github.event.inputs.setting == 'home'
+      run: npm run build:home
```

Comment:
remove the env about 30 lines up.  instead of having an if here, can you interpolate the setting into the run command?